packages/botonic-cli/src/commands/deploy.ts

```diff
@@ -3,15 +3,17 @@ import { prompt } from 'inquirer'
 import * as colors from 'colors'
 
 const fs = require('fs')
+const fsExtra = require('fs-extra')
```

Comment:
Not really related to this line but rather whole block - notice that there are `import` statements above it, but below that there are `require`s.... I think it should be consistent (you can use imports everywhere - it's TypeScript), but there are a lot of other code quality issues that linter would report so I'm not sure if you should be handling that or not.